src/odemis/driver/npmc.py

```diff
@@ -41,12 +41,31 @@
     'in': 4,
     }
 
+# Constants for referencing
+REF_POSITIVE_LIMIT = 3
+REF_NEGATIVE_LIMIT = 4
+REF_POSITIVE_LIMIT_INDEX = 5
```

Comment:
Honestly - I wasn't even sure what the difference was since it's not really explained in the docs. I just used the same name that was given in the doc. However, since the Limit & index search i snot used, maybe I can delete it.